src/scenes/dashboard/dashboard.utils.tsx

```diff
@@ -3,7 +3,7 @@ import { LineChartWrapper } from "../../components/componentUtils";
 import PieChart from "../../components/PieChart";
 import StatBox from "../../components/StatBox";
 
-export default function getCardComponentByType(type: string) {
+export default function useCardComponentByType(type: string) {
```

Comment:
If this isn't a custom we shouldn't use the name "use" on the front at least to avoid confusion.

Reply:
fixed